docs/core/tools/dotnet-publish.md

```diff
@@ -182,11 +180,9 @@ For more information, see the following resources:
 
 [!INCLUDE [os](../../../includes/cli-os.md)]
 
-- **`--sc|--self-contained [true|false]`**
-
-  Publishes the .NET runtime with your application so the runtime doesn't need to be installed on the target machine. Default is `true` if a runtime identifier is specified and the project is an executable project (not a library project). For more information, see [Self-contained deployment](../deploying/index.md#self-contained-deployment).
+- **`--sc|--self-contained`**
```

Comment:
There are quite a few arguments that are shared across a lot of commands. You could move these into includes and reference them like I did over in Aspire:

Command using an include argument: https://github.com/dotnet/docs-aspire/blob/main/docs/cli-reference/aspire-exec.md#options
Include: https://github.com/dotnet/docs-aspire/blob/main/docs/cli-reference/includes/option-project.md